src/lib/components/DeckGLMap/layers/wells/wellsLayer.ts

```diff
@@ -49,9 +60,16 @@ export default class WellsLayer extends CompositeLayer<
             this.getSubLayerProps(properties)
         );
 
-        if (this.props.outline) return [outline, colors];
-        else return [colors];
+        // Highlight the selected well.
+        properties.data = this.props.selectedFeature;
+        properties.pointRadiusScale = properties.pointRadiusScale + 2;
+        properties.lineWidthScale = properties.lineWidthScale + 2;
+        const highlight = new GeoJsonLayer<WellDataType>(properties);
```

Comment:
You need to do `this.getSubLayerProps(properties)`, otherwise the inherited props will not get propagated (e.g. opacity).